src/client/components/Meal.js

```diff
@@ -0,0 +1,187 @@
+import React, { useState, useEffect } from "react";
+import { useParams, Link } from "react-router-dom";
+import Reviews from "./Reviews";
+
+const Meal = () => {
```

Comment:
Same insight as in the PR before - a lot is happening here. Maybe some parts can be moved out? 💡